plot_types/A_basic/e_step.py

```diff
@@ -0,0 +1,25 @@
+"""
+=====================
+step(x, y, where=...)
```

Comment:
We want to have `stairs()` as well. Two alternatives:

- name this "step-like" and show both here
- or make a separate entry and link between the two.

Reply:
As above, we can add more, or modify as wanted after this is merged.